pkg/plugins/golang/v3/scaffolds/internal/templates/makefile.go

```diff
@@ -121,7 +121,7 @@ test: manifests generate fmt vet envtest ## Run tests.
 ##@ Build
 
 .PHONY: build
-build: generate fmt vet ## Build manager binary.
+build: manifests generate fmt vet ## Build manager binary.
```

Comment:
could you revert this change? It was done in another PR

Reply:
Done.